bin/doctest

```diff
@@ -11,7 +11,15 @@ from __future__ import print_function
 
 # files listed here can be in unix forward slash format with paths
 # listed relative to sympy (which contains bin, etc...)
-blacklist = []
+
+# Check version of NumPy and Theano, see https://github.com/sympy/sympy/issues/15784
+# and https://github.com/Theano/Theano/pull/6671
+import numpy as np
```

Comment:
doctest should work also when numpy is not installed without raising ImportError. It might be better to add the version test as an alternative after [this line](https://github.com/sympy/sympy/blob/master/sympy/utilities/runtests.py#L702).

Reply:
Yes, I realized the problem. I have commited a new version just now, before seeing your comment. I agree that your alternative may simplify things. Thanks!

Reply:
Fortunately, I have not set up authorization yet, so I could stop the push. Will change based on your suggestion. Will also solve the issue with the class doctest being disabled always.